### ISSUE TITLE
ShareRobot Test Set

### ISSUE THREAD

**Issue by jongsh** [FlagOpen/RoboBrain2.0#17]
Thanks for your excellent work! I noticed that you've published the [ShareRobot dataset](https://huggingface.co/datasets/BAAI/ShareRobot), which is used in the paper [RoboBrain: A Unified Brain Model for Robotic Manipulation from Abstract to Concrete](https://arxiv.org/abs/2502.21257).

I see that only the training set is currently available. Would it be possible to access the test set mentioned in the paper as well?

**Comment by tanhuajie** [comment]
Thank you for the reminder. We will release the relevant evaluation dataset within a few days. Please stay tuned for the latest updates from [FlagEvalMM](https://github.com/flageval-baai/FlagEvalMM).

**Comment by tanhuajie** [comment]
Hi, @jongsh. The related PR is now under review. https://github.com/flageval-baai/FlagEvalMM/pull/84

**Comment by jongsh** [comment]
I’m truly grateful! We can now consider this issue resolved.
